src/aeser_api_encoder.erl

```diff
@@ -191,6 +192,7 @@ type2enc(oracle_query)         -> ?BASE64;
 type2enc(oracle_query_id)      -> ?BASE58;
 type2enc(oracle_response)      -> ?BASE64;
 type2enc(account_pubkey)       -> ?BASE58;
+type2enc(account_secret_key)   -> ?BASE64;
```

Comment:
I propose to use BASE64 as it is a low-level thing don't be needed to handle by users in everyday life. Though I'm fine to use BASE58.